src/core/make-widget.js

```diff
@@ -30,6 +31,9 @@ const defaultOptions = {
 
 const queryStringKeys = {
   mode: 'typeform-embed',
+  source: 'typeform-source',
```

Comment:
From [the ticket](https://jira.typeform.tf/browse/DIST-401):

> Source: Optional value. **Domain name of the site using the SDK.**

We should read the domain from `window.location` if it is not set.

Reply:
Good catch Matej. Implementing this one ASAP.